### PR TITLE
refactor: mapproxy will able to convert image type to mapproxy image format (MAPCO-2762)

### PR DESCRIPTION
**Pull request by ronenkapelian**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✔                                                                       |

mapproxy api will convert from the request the image type into image format "image/image-type" instead getting from request

### CODE REVIEW

File: src/layers/models/layersManager.ts [hunk 1]
```diff
@@ -48,9 +49,8 @@ class LayersManager {
         throw new ConflictError(`Layer name '${layerRequest.name}' is already exists`);
       }
 
-      const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, layerRequest.format);
+      const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, this.mapToTileFormat(layerRequest.format));
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:52]: separate to a different row "this.mapToTileFormat(layerRequest.format)", not as parameter inside other function

File: src/layers/models/layersManager.ts [hunk 2]
```diff
@@ -146,7 +146,7 @@ class LayersManager {
 
   public async updateLayer(layerName: string, layerRequest: ILayerPostRequest): Promise<void> {
     this.logger.info(`Update layer: '${layerName}' request`);
-    const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, layerRequest.format);
+    const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, this.mapToTileFormat(layerRequest.format));
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:149]: separate to a different row "this.mapToTileFormat(layerRequest.format)", not as parameter inside other function

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good job!

File: src/layers/models/layersManager.ts (hunk 1 above)
Comment (ronenkapelian) [reply src/layers/models/layersManager.ts]: done

File: src/layers/models/layersManager.ts (hunk 2 above)
Comment (ronenkapelian) [reply src/layers/models/layersManager.ts]: done

**Review by asafMasa — APPROVED** [review]

**Pull request merged**